### PR TITLE
[Clean Code]move lookup_embedding from worker to embedding layer

### PR DESCRIPTION
**Pull request by QiJune**
Lookup embedding is a method of embedding layer. This PR aims to make code of customized layer, such as embedding layer,  follow the principle `low coupling and high cohesion`.

### CODE REVIEW

File: elasticdl/python/worker/worker.py [hunk 1]
```diff
@@ -126,10 +124,7 @@ def _init_embedding_layer(self):
         """
         self._embedding_layers = find_layer(self._model, Embedding)
         for layer in self._embedding_layers:
-            layer.set_lookup_func(self.lookup_embedding)
-        if self._embedding_layers:
-            # TODO check that Redis IP/PORT is set
```
Comment (mhaoli) [inline elasticdl/python/worker/worker.py:131]: Perhaps do not remove this TODO?

File: elasticdl/python/elasticdl/layers/embedding.py [hunk 2]
```diff
@@ -44,6 +48,7 @@ def __init__(
         self.supports_masking = mask_zero
         self.input_length = input_length
         self.combiner = combiner
+        self.embedding_service_endpoint = (embedding_service_endpoint,)
```
Comment (mhaoli) [inline elasticdl/python/elasticdl/layers/embedding.py]: Why do we need to convert `embedding_service_endpoint` to a tuple?

File: elasticdl/python/tests/worker_test.py [hunk 3]
```diff
@@ -160,53 +131,6 @@ def test_embedding_layer(self):
         )
         self.assertTrue(len(worker._embedding_layers) == 2)
 
-    def test_lookup_embedding(self):
```
Comment (mhaoli) [inline elasticdl/python/tests/worker_test.py:163]: Formerly, this test case is for the correctness of `worker.lookup_embedding`. Maybe do not delete it but moving it to `layer_test.py`?

File: elasticdl/python/worker/worker.py (hunk 1 above)
Comment (QiJune) [reply elasticdl/python/worker/worker.py:131]: The new logic is that

```python
for layer in self._embedding_layers:
    layer.set_endpoint(self._embedding_service_endpoint)
```
It set redis IP/Port to embedding layer directly.

File: elasticdl/python/tests/worker_test.py (hunk 3 above)
Comment (QiJune) [reply elasticdl/python/tests/worker_test.py:163]: It seems that former code compares the same thing:

```python
e_lookup, e_unknown = EmbeddingService.lookup_embedding(
   keys=["-".join([layer_name, str(id)]) for id in ids]
)
lookup_result = worker.lookup_embedding(
   ids=ids,
   layer_name=layer_name,
   embedding_table_dim=embedding_table_dim,
)
```

`worker.lookup_embedding` is a thin wrapper of `EmbeddingService.lookup_embedding`. It directly calls `EmbeddingService.lookup_embedding`. I think it's no need to make an unittest for these two functions.

File: elasticdl/python/elasticdl/layers/embedding.py (hunk 2 above)
Comment (QiJune) [reply elasticdl/python/elasticdl/layers/embedding.py]: Done, remove it.

**Review by mhaoli — APPROVED** [review]

**Pull request merged**